project/plugins.sbt

```diff
@@ -1,13 +1,11 @@
 resolvers += "bintray-spark-packages" at "https://dl.bintray.com/spark-packages/maven/"
 
-addSbtPlugin("org.spark-packages" % "sbt-spark-package" % "0.2.5")
```

Comment:
we dont need spark packages anymore?

Reply:
I was having trouble getting it working with sbt 1.x. This project only uses spark-core so that was added in the build.sbt file. I was following what [telemetry-streaming](https://github.com/mozilla/telemetry-streaming) does in their project.